### DISCUSSION TITLE
List of formats?

### DISCUSSION THREAD

**Discussion by Subtitle-Bob** [SubtitleEdit/subtitleedit#7725] [Q&A]
There are now over 100 "unknown" formats. Does anyone have a list of what the different formats produce? 

Background: I am looking for a format that outputs plain text without timecodes. Yes, I can export the subtitles as "Plain Text", but unfortunately this format does not appear under "formats", so I cannot apply it directly to a batch processing of Faster-Whisper.

**Comment by darnn** [comment]
I'm not quite sure what you're doing. What is the batch you're running? SE's batch function? Or simply processing a list of files in `Audio to text`?
If it's the latter, you may want to consider just using the command line version of Faster-Whisper. You have the option for plaintext output there, IIRC. 
If it's the former, you can select `Plain text` under `Format` in the batch convert window.

**Comment by Subtitle-Bob** [comment]
Yes, I'm processing a list of mp4 into text via faster-whisper. I know that I can use the command line of faster-whisper, but I was wondering if there is also a smarter solution via SE. 
Is there a way to implement a custom output format in SE that is displayed under `Format` like the hundred other `unknown` formats?

**Comment by darnn** [comment]
Not that I can see. There are custom formats for exporting, but they don't appear in the format list in a way that would let you use them with Whisper. Frankly, I think a plaintext format would be good to have in the list of formats too, I'm just not sure how SE would handle a format with no timecodes at all.

**Reply by Subtitle-Bob** [reply]
Should I open a request for this?

**Comment by darnn** [comment]
I'm not the one to ask, but might as well, I suppose.
